bin/gpuarray-cache

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+
+import os
+
+def clean(max_size):
+    content = []
+    for root, dirs, files in os.walk(os.environ.get('GPUARRAY_CACHE',
+                                                    '~/.gpuarray/cache/')):
```

Comment:
It should work well on mac too, it's windows I have more doubts about.

Also, maybe there should be a default, I just didn't put one in the C code.